src/main/java/Univus.java

```diff
@@ -0,0 +1,229 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Univus {
+    private static Scanner scanner;
+    private static List<Task> store;
+    public Univus() {
+        this.scanner = new Scanner(System.in);
+        this.store = new ArrayList<>();
+    }
+    public void greet() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Univus");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+    public void bye() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+    public void echo(String message) throws UnivusException {
+        if (message.equals("bye")){
+            bye();
+            scanner.close();
+        } else if (message.equals("list")) {
+            System.out.println("____________________________________________________________");
+            int index = 1;
+            for (Task msg : store) {
+                System.out.println(index + "." + msg.toString());
+                index++;
+            }
+            System.out.println("____________________________________________________________");
+        } else if (message.matches("mark\\s\\d+")) {
+            try {
+                int index = Integer.parseInt(message.split(" ")[1]);
+
+                if (index > store.size() || index < 1) {
+                    throw new UnivusException("OOPS!!! Wrong Index!");
+                }
+                Task task = store.get(index - 1);
+                if (task.isDone()) {
```

Comment:
I appreciate how you handled the situation where users should be unable to mark the task as done when it is already marked as done, same goes for unmark